security/voters.rst

```diff
@@ -48,15 +48,75 @@ which makes creating a voter even easier::
         abstract protected function voteOnAttribute(string $attribute, mixed $subject, TokenInterface $token): bool;
     }
 
-.. _how-to-use-the-voter-in-a-controller:
 
-.. tip::
+The Cacheable Voter Interface
```

Comment:
Can you target 5.4 branch instead 7.1 ? 

It's latest maintained branch and feature has been introduced here